src/main/java/org/sagebionetworks/web/client/presenter/EntityPresenter.java

```diff
@@ -331,24 +320,23 @@ public Widget asWidget() {
 
   @EventHandler
   public void onEntityUpdatedEvent(EntityUpdatedEvent event) {
-    List<SynapseReactClientEntityQueryKey> queryKey =
-      SynapseReactClientEntityQueryKey.create(
-        QueryKeyConstants.ENTITY,
-        event.getEntityId()
-      );
-    queryClient.resetQueries(queryKey);
+    KeyFactory keyFactory = new KeyFactory(
+      authenticationController.getCurrentUserAccessToken()
+    );
+
+    List<?> queryKey = keyFactory.getEntityQueryKey(event.getEntityId());
+    queryClient.invalidateQueries(queryKey);
     globalAppState.refreshPage();
   }
 
   @EventHandler
   public void onDownloadListUpdatedUpdatedEvent(
     DownloadListUpdatedEvent _event
   ) {
-    List<String> queryKey = Arrays.asList(
-      authenticationController.getCurrentUserAccessToken(),
-      QueryKeyConstants.DOWNLOAD_LIST
+    KeyFactory keyFactory = new KeyFactory(
+      authenticationController.getCurrentUserAccessToken()
     );
-    queryClient.invalidateQueries(queryKey);
+    queryClient.invalidateQueries(keyFactory.getDownloadListBaseQueryKey());
   }
```

Comment:
This invalidation recently broke when we changed `KeyFactory` to use the token hash instead of the token itself.